patches/chromium/feat_add_streaming-schemes_registry.patch

```diff
@@ -0,0 +1,135 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Paul Frazee <pfrazee@gmail.com>
+Date: Sat, 4 Apr 2020 16:27:45 -0500
+Subject: feat: add streaming-schemes registry
+
+blink::WebMediaPlayerImpl - which provides the <video> and <audio> behaviors - needs to know
+whether a data source will stream or fully buffer the response. It determines this behavior
+with MultibufferDataSource::AssumeFullyBuffered() which has http/s hardwired. An incorrect
+determination will cause the video/audio to fail playing.
+
+This patch adds a list of "streaming protocols" to the url SchemeRegistry in order to allow
+other protocols to register their streaming behavior. MultibufferDataSource::AssumeFullyBuffered()
+then refers to the registry so that it can correctly determine the data source's settings.
```

Comment:
Should also provide a way to register via `ContentClient::AddAdditionalSchemes` https://source.chromium.org/chromium/chromium/src/+/master:content/public/common/content_client.h;l=117-148 so that we can register the scheme in `ElectronContentClient::AddAdditionalSchemes` that will help the network service process understand the scheme.

Reply:
Done